src/imports.rs

```diff
@@ -39,10 +40,10 @@ impl Rewrite for ast::ViewPath {
                 let path_str = try_opt!(path.rewrite(context, width - ident_str.len() - 4, offset));
 
                 Some(if path.segments.last().unwrap().identifier == ident {
-                         path_str
-                     } else {
-                         format!("{} as {}", path_str, ident_str)
-                     })
+                        path_str
```

Comment:
Ah yes, I see! It's actually fine then. I couldn't tell it wasn't formatting in visual mode because the two only differ by a single whitespace ;p